tools/cldr-code/src/main/java/org/unicode/cldr/test/CLDRTest.java

```diff
@@ -922,10 +925,13 @@ public void TestMinimalLocalization() throws IOException {
                                     new String[] {
                                         "sun", "mon", "tue", "wed", "thu", "fri", "sat"
                                     }));
-            for (int i = -7; i < 0; ++i) {
-                checkForItems(item, (i < -4 ? months : days), i, missing, failureCount, null);
-            }
-
+            checkForItemsDate(item, months, NameType.TZ_EXEMPLAR, missing, failureCount);
+            checkForItemsDate(item, months, NameType.CURRENCY_SYMBOL, missing, failureCount);
+            checkForItemsDate(item, months, NameType.CURRENCY, missing, failureCount);
+            checkForItemsDate(item, days, NameType.VARIANT, missing, failureCount);
+            checkForItemsDate(item, days, NameType.TERRITORY, missing, failureCount);
+            checkForItemsDate(item, days, NameType.SCRIPT, missing, failureCount);
+            checkForItemsDate(item, days, NameType.LANGUAGE, missing, failureCount);
```

Comment:
This usage of the magic number "-7" to mean "NameType.TZ_EXEMPLAR and getDateKey", and so forth, involved some tricky arithmetic. This PR reduces the trickiness, at the expense of making the parameters explicit instead of looping.